frontend/src/app/core/main-menu/main-menu-navigation.service.ts

```diff
@@ -86,7 +86,7 @@ export class MainMenuNavigationService {
     // 1. unbind the current click functions
       .unbind('click')
       // 2. wrap each in a span that we'll use for the new click element
-      .wrapInner('<span class="ellipsis"/>')
+      .wrapInner('<span class="ellipsis" style="display: flex; align-items: center"/>')
```

Comment:
I'm not sure about this, it'd be nicer to fix it with a class

Reply:
definitely, but ellipsis is a global class. Should we introduce a global class that enables an inline-flex container?